lib/Auth/Process/SPAuthnContextClassRef.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+
+/**
+ * Filter for saving the SP AuthnContextClassRef.
```

Comment:
Add to class description and README
This filter should be configured on the SP:
- Specific for only the SP in authsources.php
- Specific for only one remote IdP in saml20-idp-remote or shib13-idp-remote